Shoko.Server/Plex/PlexHelper.cs

```diff
@@ -158,19 +158,15 @@ private Connection ConnectionCache
 
     private Dictionary<string, string> AuthenticationHeaders => new() { { "X-Plex-Token", GetPlexToken() } };
 
+    private DateTime? _lastAuthenticated = null;
+
     public bool IsAuthenticated
     {
         get
         {
-            if (isAuthenticated is true)
+            if (_lastAuthenticated is not null && DateTime.Now - _lastAuthenticated > TimeSpan.FromMinutes(30))
```

Comment:
Where are we setting this value?

Reply:
Oops, thanks for that... hope now is okay!